shared/actions/chat2/index.js

```diff
@@ -2694,6 +2725,7 @@ function* chat2Saga(): Saga.SagaGenerator<any, any> {
   yield Saga.safeTakeEvery(Chat2Gen.attachmentDownload, attachmentDownload)
   yield Saga.safeTakeEvery(Chat2Gen.attachmentsUpload, attachmentsUpload)
   yield Saga.safeTakeEvery(Chat2Gen.attachmentPasted, attachmentPasted)
+  yield Saga.safeTakeEvery(Chat2Gen.attachmentFullscreenNext, attachmentFullscreenNext)
```

Comment:
we should move to using actionToAction (in this case) and actionToPromise in simpler cases

Reply:
I'm not sure how to use it here, the action that would be returns from one of these `actionToAction` handlers depends on the result of an RPC call. That is, I need to call the RPC, get its response, and then dispatch the correct action depending on the result. I'm not sure how to do that in something like `Saga.sequentially` or anything like that that wouldn't make this code harder to read.